src/parsers/MetadataParser.ts

```diff
@@ -75,7 +86,8 @@ type MapMetadata = {
 
 export default class MetadataParser extends StatefulBufferParser {
   private mapmetaParser: StatefulBufferParser = new StatefulBufferParser();
-  async parse(blocks: DataBlock[]): Promise<ReplayMetadata> {
+
+  static async getUncompressedData(blocks: DataBlock[]): Promise<Buffer> {
```

Comment:
What is the reasoning of using a static method instead of an instance method here?

Reply:
It doesn't really depend on the metadata parser, it might even make sense to export it as an independent utility method?

Reply:
Yes, that is a good idea.